src/ble/att/mod.rs

```diff
@@ -17,7 +17,7 @@ use {
         uuid::{Uuid, Uuid16},
         Error,
     },
-    core::fmt,
+    core::{default::Default, fmt},
```

Comment:
This is in the prelude already, no need to import it.

Reply:
Fixed